portfolio/src/main/webapp/js/scripts.js

```diff
@@ -61,9 +61,24 @@ function createCommentElement(comment) {
     commentElement.className = 'comment';
 
     const textElement = document.createElement('span');
-    textElement.innerText = comment.text + " - " + comment.name;
+    textElement.innerText = comment.text + " - " + comment.name + "    ";
+
+    const deleteButtonElement = document.createElement('button');
+    deleteButtonElement.innerText = 'X';
+    deleteButtonElement.addEventListener('click', () => {
+        deleteComment(comment);
+        commentElement.remove();
+    });
 
     commentElement.appendChild(textElement);
+    commentElement.appendChild(deleteButtonElement);
 
     return commentElement;
+}
+
+/** Tells the server to delete the comment. */
+function deleteComment(comment) {
```

Comment:
Consider passing in just the id here if you don't need any other properties.

Reply:
done!